roles/StackStorm.nodejs/tasks/nodejs_yum.yml

```diff
@@ -10,6 +10,24 @@
   until: _task is succeeded
   tags: nodejs
```

Comment:
another thought here, probably want to add similar behavior for debian/apt so that it's consistent across OSes